serverless/serverless-config-HA.adoc

```diff
@@ -0,0 +1,11 @@
+include::modules/serverless-document-attributes.adoc[]
```

Comment:
This is an odd place to have the attribute file included. Though as I'm typing I see why, because you're using one in the title.

Reply:
Yeah, I used to have it in the normal place but it was breaking the titles :)